From 22d099f3a4da2645d1da441a7fbd0d2fa28c333f Mon Sep 17 00:00:00 2001
From: Ling Pei Lee <pei.lee.ling@intel.com>
Date: Thu, 11 Nov 2021 17:53:18 +0800
Subject: [PATCH 31/33] stmmac: intel: Update PTP clock rate from 200MHz to
 204.86MHz

Current Intel AlderLake-S and TigerLake platform has an
output of ~976ms interval when probed on 1 Pulse-per-Second(PPS)
hardware pin.

After checking with hardware team, the correct PTP clock frequency
should be 204.8MHz instead of 200MHz.

Signed-off-by: Wong Vee Khee <vee.khee.wong@linux.intel.com>
Signed-off-by: Ling Pei Lee <pei.lee.ling@intel.com>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-intel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: kernel-lts-staging/drivers/net/ethernet/stmicro/stmmac/dwmac-intel.c
===================================================================
--- kernel-lts-staging.orig/drivers/net/ethernet/stmicro/stmmac/dwmac-intel.c
+++ kernel-lts-staging/drivers/net/ethernet/stmicro/stmmac/dwmac-intel.c
@@ -756,7 +756,7 @@ static int tgl_common_data(struct pci_de
 {
 	plat->rx_queues_to_use = 6;
 	plat->tx_queues_to_use = 4;
-	plat->clk_ptp_rate = 200000000;
+	plat->clk_ptp_rate = 204800000;
 	plat->speed_mode_2500 = intel_speed_mode_2500;
 
 	plat->safety_feat_cfg->tsoee = 1;
